### PR TITLE
Walthros: Renewal

### PR DESCRIPTION
**Pull request by tabreturn**
# New Port for Walthros: Renewal

## Game Information
- **Title**: Walthros: Renewal
- **URL**: https://super-walrus-games.itch.io/walthros-renewal

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [ ] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

**Review by kloptops — DISMISSED** [review]

File: ports/walthrosrenewal/walthrosrenewal/libs.aarch64/libX11.so.6
Comment (cdeletre) [inline ports/walthrosrenewal/walthrosrenewal/libs.aarch64/libX11.so.6:1]: Are the X11 related libraries really needed ? Just asking as I haven't tested your port.

**Review by cdeletre — CHANGES_REQUESTED** [review]

File: ports/walthrosrenewal/walthrosrenewal/libs.aarch64/libX11.so.6
Comment (tabreturn) [reply ports/walthrosrenewal/walthrosrenewal/libs.aarch64/libX11.so.6:1]: doh! fixed :)

Comment (Cebion) [inline ports/walthrosrenewal/walthrosrenewal/libs.aarch64/libX11.so.6:1]: Afraid yes, if you remove them game will no longer run. I tested with removing via patchelf but it wants x11 symbols for startup check. After that it runs without using x11

**Review by Cebion — APPROVED** [review]

**Review by kloptops — APPROVED** [review]

File: ports/walthrosrenewal/port.json
[the remark this answers is not in the document]
Comment (tabreturn) [reply ports/walthrosrenewal/port.json:1]: doh! fixed

File: ports/walthrosrenewal/walthrosrenewal/libs.aarch64/libX11.so.6
Comment (tabreturn) [reply ports/walthrosrenewal/walthrosrenewal/libs.aarch64/libX11.so.6:1]: So, just leave as is?

**Pull request merged**